blocks/components/editable/index.js

```diff
@@ -54,7 +54,9 @@ export default class Editable extends wp.element.Component {
 		}
 		const value = this.editor.getContent();
 		this.editor.save();
-		this.props.onChange( value );
+		// We need the node itself to parse the content and being
+		// able to correctly swap tags.
+		this.props.onChange( value, this.node );
```

Comment:
If we find ourselves appending more arguments to this callback, we should consider if there's either a better alternative than `value` that could serve as the singular new content of the editor or if it might make sense to form this more as a proper event object with named properties.

Not clear to me if we'll need to do either of these here yet.